src/order/bounded_order.lean

```diff
@@ -1373,6 +1381,17 @@ lemma codisjoint.dual [semilattice_sup α] [order_top α] {a b : α} :
 @[simp] lemma codisjoint_of_dual_iff [semilattice_sup α] [order_top α] {a b : αᵒᵈ} :
   codisjoint (of_dual a) (of_dual b) ↔ disjoint a b := iff.rfl
 
+section distrib_lattice
+variables [distrib_lattice α] [bounded_order α] {a b c : α}
+
+lemma disjoint.le_of_codisjoint (hb : disjoint a b) (hc : codisjoint a c) : b ≤ c :=
```

Comment:
What do you think of
```suggestion
@[trans]
lemma disjoint.le_of_codisjoint (hb : disjoint a b) (hc : codisjoint b c) : a ≤ c :=
```

Reply:
What's the reasoning? Can you really use that in a `calc` block?

Reply:
I've not tried! It just looks like a slightly more natural argument order to me, but it's not a big deal. I can test it out with calc later if you'd like

Reply:
At least, you need infix notation to use it with `calc`.

Reply:
You're right, this probably doesn't matter. I still think the formulation is more natural as
```suggestion
lemma disjoint.le_of_codisjoint (hab : disjoint a b) (hbc : codisjoint b c) : a ≤ c :=
```
because that gives a clear expection of which order to expect `hab` and `hbc` and justification of why this order is more natural, but the `@[trans]` is likely pointless.